src/librustdoc/lib.rs

```diff
@@ -618,6 +621,30 @@ fn opts() -> Vec<RustcOptGroup> {
                 "Make the identifiers in the HTML source code pages navigable",
             )
         }),
+        unstable("scrape-examples-output-path", |o| {
+            o.optopt(
+                "",
+                "scrape-examples-output-path",
+                "",
+                "collect function call information and output at the given path",
+            )
+        }),
+        unstable("scrape-examples-target-crate", |o| {
```

Comment:
Hmm, why is this a separate option than --output? When would cargo need to customize it?

Reply:
This is an optimization so the generated file doesn't include *every* call site, just the ones relevant to the crates being documented (eg not stdlib). I wanted to keep the footprint of the intermediate files to a minimum.